libcxxabi/test/test_demangle.pass.cpp

```diff
@@ -30220,6 +30220,11 @@ const char* cases[][2] = {
     {"_ZZNH3Foo3fooES_iENK4Foo24foo2Ev", "Foo::foo(this Foo, int)::Foo2::foo2() const" },
     {"_ZNH3FooclERKS_", "Foo::operator()(this Foo const&)"},
 
+
+    // C++26 pack indexing
```

Comment:
We should add invalid cases as well see `invalid_cases` and make sure we cover each case that can return a `nullptr` in `AbstractManglingParser` (I think)